src/journal/journald-server.c

```diff
@@ -212,12 +212,12 @@ void server_space_usage_message(Server *s, JournalStorage *storage) {
                 return;
 
         metrics = &storage->metrics;
-        format_bytes(fb1, sizeof(fb1), storage->space.vfs_used);
-        format_bytes(fb2, sizeof(fb2), metrics->max_use);
-        format_bytes(fb3, sizeof(fb3), metrics->keep_free);
-        format_bytes(fb4, sizeof(fb4), storage->space.vfs_available);
-        format_bytes(fb5, sizeof(fb5), storage->space.limit);
-        format_bytes(fb6, sizeof(fb6), storage->space.available);
+        FORMAT_BYTES(storage->space.vfs_used);
+        FORMAT_BYTES(metrics->max_use);
+        FORMAT_BYTES(metrics->keep_free);
+        FORMAT_BYTES(storage->space.vfs_available);
+        FORMAT_BYTES(storage->space.limit);
+        FORMAT_BYTES(storage->space.available);
```

Comment:
Something went wrong here. I think these expressions need to be substituted into the `server_driver_message()` function call below. Also, spurious lines in the commit message.

Reply:
Oopsie. Thanks for catching this.

I added follow-up commits to re-enable `-Wunused-result` and decorate the formatting functions with `__attribute__((__warn_unused_result__))`.